File: src/Entités/Sponsor.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package entities;

/**
 *
 * @author rahma
 */
public class Sponsor  {
    private int idS ;
    private String nomS ;
    private String prenomS ;
    private String descriptionS ; 
    private String imageS ;
    public Sponsor() {
        
    }

    public Sponsor(int idS, String nomS, String prenomS, String descriptionS , String imageS) {
        this.idS = idS;
        this.nomS = nomS;
        this.prenomS = prenomS;
        this.descriptionS = descriptionS;
        this.imageS=imageS ;
    }

    public Sponsor(String nomS, String prenomS, String descriptionS ,String imageS) {
        this.nomS = nomS;
        this.prenomS = prenomS;
        this.descriptionS = descriptionS;
        this.imageS=imageS ;
    }

    public String getImageS() {
        return imageS;
    }

    public void setImageS(String imageS) {
        this.imageS = imageS;
    }

    public int getIdS() {
        return idS;
    }

    public String getNomS() {
        return nomS;
    }


    public String getPrenomS() {
        return prenomS;
    }

    public String getDescriptionS() {
        return descriptionS;
    }

    public void setIdS(int idS) {
        this.idS = idS;
    }

    public void setNomS(String nomS) {
        this.nomS = nomS;
    }

    public void setPrenomS(String prenomS) {
        this.prenomS = prenomS;
    }

    public void setDescriptionS(String descriptionS) {
        this.descriptionS = descriptionS;
    }

    @Override
    public String toString() {
        return "Sponsor{" + "idS=" + idS + ", nomS=" + nomS + ", prenomS=" + prenomS + ", descriptionS=" + descriptionS +", ImageS=" + imageS + '}';
    }
    
}

File: Desktop/gestionevenement/src/Entity/Sponsor.php
<?php

namespace App\Entity;

use Doctrine\ORM\Mapping as ORM;

/**
 * Sponsor
 *
 * @ORM\Table(name="sponsor", uniqueConstraints={@ORM\UniqueConstraint(name="idS_2", columns={"idS"}), @ORM\UniqueConstraint(name="idS", columns={"idS"})})
 * @ORM\Entity
 */
class Sponsor
{
    /**
     * @var int
     *
     * @ORM\Column(name="idS", type="integer", nullable=false)
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="IDENTITY")
     */
    private $ids;

    /**
     * @var string
     *
     * @ORM\Column(name="prenomS", type="string", length=255, nullable=false)
     */
    private $prenoms;

    /**
     * @var string
     *
     * @ORM\Column(name="descriptionS", type="string", length=255, nullable=false)
     */
    private $descriptions;

    /**
     * @var string
     *
     * @ORM\Column(name="nomS", type="string", length=255, nullable=false)
     */
    private $noms;

    /**
     * @var string
     *
     * @ORM\Column(name="imageS", type="string", length=255, nullable=false)
     */
    private $images;

    public function getIds(): ?int
    {
        return $this->ids;
    }

    public function getPrenoms(): ?string
    {
        return $this->prenoms;
    }

    public function setPrenoms(string $prenoms): self
    {
        $this->prenoms = $prenoms;

        return $this;
    }

    public function getDescriptions(): ?string
    {
        return $this->descriptions;
    }

    public function setDescriptions(string $descriptions): self
    {
        $this->descriptions = $descriptions;

        return $this;
    }

    public function getNoms(): ?string
    {
        return $this->noms;
    }

    public function setNoms(string $noms): self
    {
        $this->noms = $noms;

        return $this;
    }

    public function getImages(): ?string
    {
        return $this->images;
    }

    public function setImages(string $images): self
    {
        $this->images = $images;

        return $this;
    }


}
